tests/kernel/threads/no-multithreading/boards/bt510.conf

```diff
@@ -0,0 +1,37 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Many settings in this are duplicated.  Only need to disable I2C.

Reply:
Updated to only disable I2C devices.